factory/usr/lib/systemd/system/plymouth-start.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Show Plymouth Boot Screen
+DefaultDependencies=no
+Wants=systemd-ask-password-plymouth.path systemd-vconsole-setup.service
+After=systemd-vconsole-setup.service systemd-udev-trigger.service systemd-udevd.service
+After=snap-initramfs-mounts.service
```

Comment:
This sounds quite late.
 - We should have a splash before disks are scanned.
 - If `snap-bootstrap` needs a password, it will call `systemd-ask-password`, which if plymouth is properly started, will use `systemd-ask-password-plymouth.{path,service}` (units we need in initramfs), to ask the password within plymouth.

If we need to load a theme from the gadget, I would suggest we switch theme after `snap-bootstrap`.

Reply:
As discussed, we will do this later when have the patch to switch themes in the jammy deb.

Reply:
We should still put `systemd-ask-password-plymouth.{path,service}` in the initramfs.